client/index.html

```diff
@@ -33,6 +34,16 @@ <h1 class="center">Dashboard</h1>
 					</div>
 				</div>
 				<div class="row">
+						<p>Show your QR code to check in when you get here!</p>
+						<div id="qrCode" data="registration-{{siteTitle}}-{{user._id}}"></div>
```

Comment:
just `data="{{user._id}}"` is enough, `registration` can only see data from its own event and this ID is probably not going to collide anyway.

Reply:
I talked with @petschekr  and he thought that namespacing based on event title would be better, if we have multiple registration apps and want to know what event we're referencing.

Reply:
I disagree with @petschekr , I would maybe agree if we had some unchanging `event id` but the title is meant to be human-readable. If it was just a user ID then users can use the same QR code through all our events and apps, making their lives easier. Also when they bring it to show us we will already know what event they are coming to and apps like checkin will go to the correct registration url for that event.

Reply:
We should wait for @petschekr to weigh in though.

Reply:
I'd argue that it's trivial to extract the user id from the QR code, since everything's separated by dashes, and the event name might come in handy if we ever have two events on the same day, or if someone shows a QR code from an old event at a new event; if someone who didn't apply in time for HackGT wanted to get in, they'd be able to just show their HackGTeeny QR code.

Reply:
I'm fine with it either way. The reason I initially wanted it to have that prefix was so that QR codes could be identified as being registration codes and not just random codes if read from a normal non-checkin reader